crates/ruff_python_formatter/tests/snapshots/black_compatibility@cases__comment_after_escaped_newline.py.snap

```diff
@@ -21,16 +21,16 @@ def bobtwo(): \
 ```diff
 --- Black
 +++ Ruff
-@@ -1,6 +1,8 @@
+@@ -1,8 +1,8 @@
 -def bob():  # pylint: disable=W9016
 +def bob():
 +    # pylint: disable=W9016
      pass
  
  
--def bobtwo():  # some comment here
-+def bobtwo():
-+    # some comment here
+ def bobtwo():
+-
```

Comment:
Ruff does not preserve empty lines between the class and any leading comment (intentional decision)